src/Symfony/Bundle/TwigBundle/TwigEngine.php

```diff
@@ -51,7 +51,8 @@ public function setDefaultEscapingStrategy($strategy)
     }
 
     /**
-     * @deprecated Deprecated since version 2.3, to be removed in 3.0. Use TwigDefaultEscapingStrategy instead.
+     * @deprecated since version 2.3, to be removed in 3.0.
+     *             Use TwigDefaultEscapingStrategy instead.
      */
     public function guessDefaultEscapingStrategy($filename)
```

Comment:
please add a warning here (TwigBundle does not use the deprecated method anymore.